GoogleForCreators/web-stories-wp
# Context Menu: Clear text styles

## Context

Clearing styles from text elements wasn't enabled.

## Summary

Enable clearing styles from text elements.

## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Insert a text element into the canvas
2. Right click the text element
3. Select `Clear Text Styles`. Text should be reset to 'default styles' (not specific to preset. All text elements will be reset to this configuration)
4. Add text or revert to original style
5. Select multiple elements of different types
6. Right click on the frame and select `Clear Styles`
7. Text element (and other elements) should be reverted to their style defaults.


## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #10018
